### PR TITLE
README.md Update

### LINKED ISSUE #48: There should be no spelling errors in the README
**Is your feature request related to a problem? Please describe.**
Minor to major problems with spelling

**Describe the solution you'd like**
Better English skills

**Additional context**
I was asked by a colleague to fix this

### PR DESCRIPTION
**Pull request by IgnemCC**
**Checklist:**

 
 - [ ] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [ ] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there are some spelling errors in The README. #48 

**What is the new behavior (if this is a feature change)?**
- Now all spelling errors has been fixed in The README. Closes #48

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -222,18 +223,18 @@ console.log(data.username, data.accessToken)
 ## [](#contributing)Contributing
 ![GitHub last commit](https://img.shields.io/github/last-commit/snek-at/intel) ![GitHub issues](https://img.shields.io/github/issues-raw/snek-at/intel) ![GitHub closed issues](https://img.shields.io/github/issues-closed-raw/snek-at/intel?color=green)
 
-Please read through our [contributing guidelines](https://github.com/snek-at/front/blob/master/CONTRIBUTING.md). Included are directions for opening issues, coding standards, and notes on development.
+Please read through our [guidelines for contribution](https://github.com/snek-at/front/blob/master/CONTRIBUTING.md). Included are directions for opening issues, coding standards, and notes on development.
```
Comment (schettn) [inline README.md:226]: I would change this back to "contributing guidelines"

Comment (kleberbaum) [reply README.md:226]: fair

Comment (kleberbaum) [reply README.md:226]:
```suggestion
Please read through our [contributing guidelines](https://github.com/snek-at/front/blob/master/CONTRIBUTING.md). Included are directions for opening issues, coding standards, and notes on development.
```

File: README.md [hunk 2]
```diff
@@ -7,7 +7,8 @@
 
 <h3 align="center">SNEKNetwork for Engineers and Knowledged</h3>
 <p align="center">
-The SNEK project is an attempt to create a transparent, open-source non-profit platform that allows engineers to categorize and compare. It should enable engineers from adjacent fields to visualize each other's skills through visualization and project identification.
+The SNEK project is an attempt to create a transparent, open-source, non-profit platform that enables engineers to categorize and compare themselves with other software developers in their respective fields, through project identification and visialization.
```
Comment (kleberbaum) [inline README.md:10]: I think the old description was better.

Comment (kleberbaum) [reply README.md:10]:
```suggestion
The SNEK project is an attempt to create a transparent, open-source non-profit platform that allows engineers to categorize and compare. It should enable engineers from adjacent fields to visualize each other's skills through visualization and project identification.
```

File: README.md [hunk 3]
```diff
@@ -7,7 +7,7 @@
 
 <h3 align="center">SNEKNetwork for Engineers and Knowledged</h3>
 <p align="center">
-The SNEK project is an attempt to create a transparent, open-source, non-profit platform that enables engineers to categorize and compare themselves with other software developers in their respective fields, through project identification and visialization.
+The SNEK project is an attempt to create a transparent, open-source non-profit platform that allows engineers to categorize and compare. It should enable engineers from adjacent fields to visualize each other's skills through visualization and project identification.
```
Comment (Aichnerc) [inline README.md:10]: Missing comma bro.
```suggestion
The SNEK project is an attempt to create a transparent, open-source, non-profit platform that allows engineers to categorize and compare. It should enable engineers from adjacent fields to visualize each other's skills through visualization and project identification.
```

Comment (Aichnerc) [reply README.md:10]: Nevermind. I was proven wrong here brotha.

**Pull request merged**